gio/src/subclass/initable.rs

```diff
@@ -0,0 +1,177 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::object::Cast;
+use glib::translate::*;
+use glib::Error;
+
+use glib::subclass::prelude::*;
+
+use std::ptr;
+
+use crate::Cancellable;
+use crate::Initable;
+
+pub trait InitableImpl: ObjectImpl + Send {
```

Comment:
```suggestion
pub trait InitableImpl: ObjectImpl {
```

`Initable` does not require the object to be send'able (and GObjects are never only `Send`, if anything they must be `Send + Sync` due to refcounting).